src/foremast/pipeline/construct_pipeline_block.py

```diff
@@ -189,6 +189,11 @@ def ec2_pipeline_setup(appname='', project='', settings=None, env='', region='',
     instance_security_groups.append(appname)
     instance_security_groups.extend(settings['security_group']['instance_extras'])
 
+    for each_sg, duplicate_sg_name in SECURITYGROUP_REPLACEMENTS.items():
```

Comment:
Copying code means this should be a function.

Reply:
Done. Moved to utils.